src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAXNUMBER = 100;
+    private Random number = new Random();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setNumber(getRandomNumber());
+        ball.setColor(String.valueOf(colorSupplier.getRandomColor()));
+        return ball;
+    }
+
+    private int getRandomNumber() {

```

Comment:
let's remove this method, and use logic directly inside of a method where it needs